defaults/defaults_unix.go

```diff
@@ -34,4 +34,6 @@ const (
 	DefaultFIFODir = "/run/containerd/fifo"
 	// DefaultRuntime is the default linux runtime
 	DefaultRuntime = "io.containerd.runc.v2"
+	// DefaultOOMScore is the default OOM score for the containerd daemon
+	DefaultOOMScore = -999
```

Comment:
`-999` is good! that's what we use in other places
- https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/ansible/roles/containerd/templates/etc/systemd/system/containerd.service.d/memory-pressure.conf#L8
- https://github.com/kubernetes-sigs/image-builder/issues/112

Reply:
One thing I'm wondering is if it's better to set the OOM score this way, or better to go back to setting it in the systemd-unit; does cgroupv2 make a difference here? (and is is better for that to have systemd set the OOM score?)

/cc @AkihiroSuda